docs/oauth2.md

```diff
@@ -0,0 +1,30 @@
+Oauth2
+======
+
+Oauth2 is a django app that will can be used to cache your `django-oauth-toolkit`
+access token model.
```

Comment:
I just remembered why this is so. The `shortcuts.get_oauth2_app` needs `django-oauth-toolkit`. I think we will need it back in dev-requirements. Sorry.